packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -1,165 +1,306 @@
+    warning: Pick<PaletteRange, BasePaletteRange>;
+    text: Pick<PaletteText, 'primary' | 'secondary' | 'tertiary'>;
+    background: Pick<PaletteBackground, 'default' | 'level1' | 'level2' | 'level3'>;
+    focusVisible: Palette['focusVisible'];
   };
-  borderRadius: Pick<BorderRadius, 'md'>;
-  htmlFontSize: React.CSSProperties['fontSize'];
+  borderRadius: Pick<BorderRadius, 'default' | 'xs' | 'sm' | 'md' | 'lg' | 'xl'>;
+  elevationRing: CSSProperties['boxShadow'];
+  elevation: Pick<Elevation, 'xs' | 'sm' | 'md' | 'lg' | 'xl'>;
+  focus: Pick<Focus, 'default'>;
   fontSize: Pick<
     FontSize,
-    'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xl2' | 'xl3' | 'xl4' | 'xl5' | 'xl6'
-  >;
-  fontFamily: Pick<FontFamily, 'sans' | 'mono'>;
-  fontWeight: Pick<
-    FontWeight,
-    'light' | 'regular' | 'medium' | 'semiBold' | 'bold' | 'extraBold' | 'black'
+    'default' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xl2' | 'xl3' | 'xl4' | 'xl5' | 'xl6'
   >;
-  lineHeight: Pick<LineHeight, 'xs' | 'sm' | 'normal' | 'lg'>;
-  letterSpacing: Pick<LetterSpacing, 'xs' | 'normal' | 'lg'>;
+  fontFamily: Pick<FontFamily, 'default' | 'display' | 'code' | 'fallback'>;
+  fontWeight: Pick<FontWeight, 'default' | 'xs' | 'sm' | 'md' | 'lg' | 'xl'>;
+  lineHeight: Pick<LineHeight, 'default' | 'sm' | 'md' | 'lg'>;
+  letterSpacing: Pick<LetterSpacing, 'default' | 'sm' | 'md' | 'lg'>;
   typography: Pick<
-    TypographySystems,
-    | 'h1'
-    | 'h2'
-    | 'h3'
-    | 'h4'
-    | 'h5'
-    | 'headingSubtitle'
-    | 'body'
-    | 'caption'
-    | 'detail'
-    | 'headingIntro'
-    | 'button'
-    | 'overline'
+    TypographySystem,
+    'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'body1' | 'body2' | 'body3'
   >;
-  shadow: Pick<Shadow, 'ring' | 'md'>;
+  variants: Pick<Variants, DefaultVariantKey> &
+    Record<DefaultContextualOverrides, Record<Exclude<ColorPaletteProp, 'context'>, CSSObject>>;
 };
 
-/**
- * Base Joy Theme
- * Any value with `var(--joy-*)` can be used. 'joy-' will be replaced by the application prefix if provided.
- */
-const themeWithoutVars: BaseJoyTokens = {
+export const lightColorSystem: Pick<BaseJoyTokens, 'palette' | 'elevationRing'> = {
+  palette: {
+    primary: {
+      ...colors.purple,
+      ...createLightModeVariantVariables('primary'),
+    },
+    neutral: {
+      ...colors.grey,
+      ...createLightModeVariantVariables('neutral'),
+    },
+    danger: {
+      ...colors.red,
+      ...createLightModeVariantVariables('danger'),
+    },
+    info: {
+      ...colors.blue,
+      ...createLightModeVariantVariables('info'),
+    },
+    success: {
+      ...colors.green,
+      ...createLightModeVariantVariables('success'),
+    },
+    warning: {
+      ...colors.yellow,
+      ...createLightModeVariantVariables('warning'),
+    },
+    text: {
+      primary: 'var(--joy-palette-neutral-800)',
+      secondary: 'var(--joy-palette-neutral-600)',
+      tertiary: 'var(--joy-palette-neutral-500)',
```

Comment:
```suggestion
      secondary: 'var(--joy-palette-neutral-700)',
      tertiary: 'var(--joy-palette-neutral-600)',
```

A little bump on these.

Reply:
Are you sure? I still think the `tertiary` should be `500` because the contrast still works with `neutral.100`

![image](https://user-images.githubusercontent.com/18292247/143969677-dfb9b547-42a3-4de9-90fd-97b00f390f99.png)

changing `secondary` to `700` is too close to the `primary` in my opinion.

I think we should go with the existing one and try it out first, if you think this does not work we can change it before the first release.